### ISSUE TITLE
Docs/Guides link to solution which is different

### ISSUE THREAD

**Issue by jedla97** [quarkusio/quarkus#34751]
### Describe the bug

In some hibernate guides is stated that solution is located in the quickstart but this solution is totally different then what guide describing (even classes are different). Here are wrong docs which I found when I go trough it:

- [hibernate-orm-panache-kotlin-quickstart](https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/hibernate-orm-panache-kotlin.adoc) and [link to quickstart](https://github.com/quarkusio/quarkus-quickstarts/tree/main/hibernate-orm-panache-kotlin-quickstart)
- [hibernate-orm-panache](https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/hibernate-orm-panache.adoc) and [link to quickstart](https://github.com/quarkusio/quarkus-quickstarts/tree/main/hibernate-orm-panache-quickstart)
- [hibernate-orm](https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/hibernate-orm.adoc) and [link to quickstart](https://github.com/quarkusio/quarkus-quickstarts/tree/main/hibernate-orm-quickstart)
- [hibernate-reactive-panache](https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/hibernate-reactive-panache.adoc) and [link to quickstart](https://github.com/quarkusio/quarkus-quickstarts/tree/main/hibernate-reactive-panache-quickstart)
- [hibernate-reactive](https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/hibernate-reactive.adoc) and [link to quickstart](https://github.com/quarkusio/quarkus-quickstarts/tree/main/hibernate-reactive-quickstart)

Also found the similar case for [liquibase-mongodb](https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/liquibase-mongodb.adoc) ([link to quickstart](https://github.com/quarkusio/quarkus-quickstarts/tree/main/liquibase-mongodb-quickstart))

### Additional information

In my point of view there isn't necessary need to change quickstart or guide but simply change the wording from `The solution is located in the` to something like `Another example can be found in the`.

**Comment by gsmet** [comment]
Yeah, we should probably change the wording in the guides for now as these guides have not been written with a quickstart in mind. I'll have a look.

At some point, they will be rewritten but that's not something we will do now.
